test/e2e/framework/util.go

```diff
@@ -3252,10 +3252,6 @@ func NodeAddresses(nodelist *v1.NodeList, addrType v1.NodeAddressType) []string
 	hosts := []string{}
 	for _, n := range nodelist.Items {
 		for _, addr := range n.Status.Addresses {
-			// Use the first external IP address we find on the node, and
-			// use at most one per node.
-			// TODO(roberthbailey): Use the "preferred" address for the node, once
-			// such a thing is defined (#2462).
```

Comment:
Comment provided no value;
 - was incorrect about looking for external IPs
 - was duplicating description of the method when stating it returned the first value
 - the TODO is about 3 years old and refers to a closed issue